network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -129,6 +129,9 @@ class Signup(TranslatableMixin, CTA):
         # Fields from the CTA model
```

Comment:
What do you think about also adding the `name` field here? 


Also, should we set the `translatable_fields` also on the `CTA` model? And only extend that list in the `Signup` model?

Reply:
Hi @tbrlpld! This also makes sense to me. To implement the following suggestion, I have updated the `CTA` model to include the following translatable_fields:

```python
class CTA(models.Model):
...
    translatable_fields = [
        TranslatableField("name"),
        TranslatableField("header"),
        TranslatableField("description"),
        SynchronizedField("newsletter"),
    ]
```

and am extending the list in the Signup Model like so:

```python
class Signup(TranslatableMixin, CTA):
...
    translatable_fields = CTA.translatable_fields + [
        SynchronizedField("campaign_id"),
        SynchronizedField("ask_name"),
    ]
```

Wanted to double check this is what you were suggesting with your comment before I merge this in? Thanks!

Reply:
Yup @danielfmiranda That's exactly what I meant.